### PR TITLE
Fixes #732. Update sys.path to fix server spawning issue.

### LINKED ISSUE #732: Automatic server re-spawning fails if plugin updates serverRoot
In that particular case, I am assuming this is due to the server root swaping in [girder/covalic/server/__init__.py#L160](https:/github.com/girder/covalic/blob/0c831d200396ad5f69b3c11257f416dd151c3446/server/__init__.py#L160)

Wonder if there is anything that could be done 

```
[11/Mar/2015:11:01:23] ENGINE Re-spawning /home/jcfr/Projects/girder/girder/__main__.py
Traceback (most recent call last):
  File "/home/jcfr/Projects/girder/girder/__main__.py", line 23, in <module>
    from girder.utility import server  # pragma: no cover
ImportError: No module named girder.utility
```

### PR DESCRIPTION
**Pull request by jcfr**
Inspired-by: Zach Mullen <zach.mullen@kitware.com>

### CODE REVIEW

File: girder/__main__.py [hunk 1]
```diff
@@ -19,9 +19,19 @@
...
-
+import os  # pragma: no cover
+import sys  # pragma: no cover
+
+try:  # pragma: no cover
+    from girder.utility import server
+except ImportError:
+    # Update python path to ensure server respawning works. See #732
+    source_root_dir = os.path.join(os.path.dirname(__file__), '..')
+    import sys
```
Comment (zachmullen) [inline girder/__main__.py]: You already have imported sys above

**Comment by zachmullen** [comment]
This breaks the `python_static_analysis_core` test.

**Comment by jcfr** [comment]
```
$ ctest -R python_static_analysis_core
Test project /home/jcfr/Projects/girder-build
    Start 8: python_static_analysis_core
1/2 Test #8: python_static_analysis_core .........   Passed    4.51 sec
    Start 9: python_static_analysis_core_tests
2/2 Test #9: python_static_analysis_core_tests ...   Passed    3.82 sec

100% tests passed, 0 tests failed out of 2

Total Test time (real) =   8.34 sec
```

**Comment by jcfr** [comment]
@zachmullen Et voila. After n attempts .. the travis vm stopped crashing.

**Comment by zachmullen** [comment]
LGTM. Travis does tend to have better results for us after normal business hours...

**Pull request merged**